src/cli/start.js

```diff
@@ -49,43 +62,37 @@ module.exports = function(projectPath, options) {
     const bf = eval('require')(botfile)
     const dataDir = util.getDataLocation(bf.dataDir, projectPath)
     const modulesConfigDir = util.getDataLocation(bf.modulesConfigDir, projectPath)
-    return [
-      dataDir,
-      modulesConfigDir,
-      'node_modules'
-    ]
+    return [dataDir, modulesConfigDir, 'node_modules']
   }
 
   const opts = options.opts()
   if (opts.watch || opts.w) {
     util.print('info', '*** watching files for changes ***')
 
     const argvWithoutWatch = process.argv.filter(arg => !/^(--watch|-w)$/.test(arg))
+    argvWithoutWatch[0] = getPath(argvWithoutWatch[0])
+
     const nodemonOptions = {
       cwd: process.cwd(),
       exec: argvWithoutWatch.join(' '),
       ext: opts.watchExt,
-      watch: (opts.watchDir && opts.watchDir.length) ? opts.watchDir : undefined,
```

Comment:
Hm, I think `prettier` is putting these parens and wouldn't let you commit without them.
Same for some formatting below.
We have the precommit hook that should have changed that, did you commit without running `npm install` (`npm test`)?
We should also enforce these checks in our CI.

Reply:
@emirotin the parens are in the code, but they are being removed by `prettier` (I guess)

I did run `npm test`
results: `101 passing (6s)`

Reply:
That's strange, I thought prettier is the one who added them, but it might have change between its versions or I may have false memories :)

Reply:
here is the code from my branch:
```
      watch: opts.watchDir && opts.watchDir.length ? opts.watchDir : undefined,
      ignore: opts.watchIgnore && opts.watchIgnore.length ? opts.watchIgnore : getDefaultWatchIgnore(),
```

is this correct?

Reply:
As long as you see something like this

```
husky > npm run -s precommit (node v8.9.3)

 ✔ Running tasks for *.{js,jsx}
 ↓ Running tasks for *.{json,css,scss} [skipped]
   → No staged files match *.{json,css,scss}
```

when committing your changes all should be good :)

Reply:
yes:
```sh
husky > npm run -s precommit (node v8.9.3)

[14:31:14] Running tasks for *.{js,jsx,json,css} [started]
[14:31:14] Running tasks for *.{js,jsx,json,css} [skipped]
[14:31:14] → No staged files match *.{js,jsx,json,css}
On branch issue-313-win32-start-watch
Changes not staged for commit:
        modified:   src/cli/start.js

```

Reply:
Hey, just wanted to confirm that prettier passes on `master` but not `develop/x`. I think the rules are slightly different there. Line 12 of this file contains a `var` declaration which is now prevented.